components/ui/archive.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react';
+import ApiService from '../../utils/api_service'; // Assuming the correct path
+import PropTypes from 'prop-types';

```

Comment:
Make sure you import the note page you're using since the delete button is on it.